website/project/metadata/erpc-prize.json

```diff
@@ -0,0 +1,151 @@
+            "required": false
+        }]
+    }, {
+        "id": "page3",
+        "title": "Analysis Plan",
+        "description": "<p>You may describe one or more analyses in this preregistration. Please remember that all analyses specified below must be reported in the final article, and any additional analyses must be noted as exploratory or hypothesis generating.</p><p> An analysis plan must state up front which variables are predictors (independent) and which are the outcomes (dependent), otherwise it is an exploratory analysis. You are allowed to describe any exploratory work here, but a clear confirmatory analysis is required.</p>",
+        "questions": [{
+            "qid": "q7",
+            "nav": "Statistical models",
+            "type": "string",
+            "format": "textarea",
+            "title": "Statistical models",
+            "description": "What statistical model will you use to test each hypothesis? Please include the type of model (e.g. ANOVA, multiple regression, SEM, etc) and the specification of the model (this includes each variable that will be included as predictors, outcomes, or covariates). Please specify any interactions that will be tested and remember that any test not included here must be noted as an exploratory test in your final article.",
+            "required": true
+            }, {
+            "qid": "q8",
+            "nav": "Transformations",
+            "type": "string",
+            "format": "textarea",
+            "title": "Transformations",
+            "description": "If you plan on transforming, centering, recoding the data, or will require a coding scheme for categorical variables, please describe that process.",
+            "required": false
+            }, {
+            "qid": "q9",
+            "nav": "Follow-up analyses",
+            "type": "string",
+            "format": "textarea",
+            "title": "Follow-up analyses",
+            "description": "If not specified previously, will you be conducting any confirmatory analyses to follow up on effects in your statistical model, such as subgroup analyses, pairwise or complex contrasts, or follow-up tests from interactions. Remember that any analyses not specified in this research plan must be noted as exploratory.",
+            "required": false
+            }, {
+            "qid": "q10",
+            "nav": "Inference criteria",
+            "type": "string",
+            "format": "textarea",
+            "title": "Inference criteria",
+            "description": "What criteria will you use to make inferences? Please describe the information youÍll use (e.g. p-values, bayes factors, specific model fit indices), as well as cut-off criterion, where appropriate. Will you be using one or two tailed tests for each of your analyses? If you are comparing multiple conditions or testing multiple hypotheses, will you account for this?",
```

Comment:
This language (everything in the schema) is still draft, and was copy/pasted directly from the ERPC design document. I'll wait until the draft is approved to make any typographical corrections.